### ISSUE TITLE
First steps for styleguide

### ISSUE THREAD

**Issue by stenskjaer** [lombardpress/lombardpress-schema#78]
Hi guys!

I have been thinking about whether we might want a styleguide. Nothing fancy really, just to agree on the inevitable problem numero uno of any collaborate development project: Tabs or spaces [play dramatic cadence here!]

So I have made a wiki page where we can document these things. For now, since I took the initiative, I also took the liberty of making some opinionated suggestions. 
**Everything is entirely up for discussion, it just wanted it to say something.**

You can see it here: https://github.com/lombardpress/lombardpress-schema/wiki/Style-guide

**Comment by nivaca** [comment]
Excellent! I agree with your suggestions.
NV

**Comment by jeffreycwitt** [comment]
yes, Nick and i we're just discussing that we need exactly this. 

We should include instructions in the Wiki on how to set up Oxygen so that these style guides will be followed automatically.

**Comment by stenskjaer** [comment]
Yes, certainly! I could also make one for Emacs.

**Comment by jeffreycwitt** [comment]
I'd like to see the style guide moved from the wiki (which is actually just another repo) to an appendix document that is included and versioned with the  lbp-schema itself.
